core/src/main/java/org/elasticsearch/index/translog/TranslogWriter.java

```diff
@@ -191,6 +205,14 @@ public int totalOperations() {
         return operationCounter;
     }
 
+    public long getMinSeqNo() {
```

Comment:
I wonder if it's possible to read these out of order and therefore get inconsistent values? like if you read max first and then min? Also I wonder if the default value can be confusing?